src/main/java/edu/kis/vh/nursery/IntArrayStack.java

```diff
@@ -0,0 +1,35 @@
+package edu.kis.vh.nursery;
+
+public class IntArrayStack {
+    public static final int MAX_LENGTH = 12;
+    public static final int EMPTY_VALUE = -1;
+    public static final int MAX_INDEX = 11;
+    private final int[] numbers = new int[MAX_LENGTH];
+
+    public int total = EMPTY_VALUE;
```

Comment:
Errors in collaborators work (lack of good review). Extracting constants is done incorrectly. Have you asked for review hints (1.4.w7): tiny.cc/998muz ?
This constant should be different (separate) than the one in peekaboo and countOut methods.